src/query.cr

```diff
@@ -260,14 +260,14 @@ module Cql
     # => "SELECT users.name, users.age, address.city, address.state FROM users, address"
     # ```
     def select(**fields)
-      fields.each do |k, v|
-        if [:count, :sum, :avg, :min, :max].includes?(k) && v.is_a?(Symbol)
-          @aggr_columns << build_aggr_expression(k, v)
+      fields.each do |key, value|
+        if [:count, :sum, :avg, :min, :max].includes?(key) && value.is_a?(Symbol)
+          @aggr_columns << build_aggr_expression(key, value)
         else
-          if v.is_a?(Array(Symbol))
-            v.map { |f| @columns << find_column(f, k) }
+          if value.is_a?(Array(Symbol))
+            value.map { |name| @columns << find_column(name, key) }
```

Comment:
Although Ameba does not recommend renaming `k` and `v`, it does suggest renaming `f`. I decided to update everything for consistency.